storage/storage.hpp

```diff
@@ -230,6 +232,9 @@ class Storage
   /// a list of available maps. They are all available countries expect for fully downloaded
   /// countries. That means all mwm of the countries have been downloaded.
   void GetCountyListToDownload(TCountriesVec & countryList) const;
+  /// \brief Calls |forEach| for each node for subtree with parent == |parent|.
+  /// For example ForEachInSubtree(GetRootId()) calls |forEach| for every node including the root.
```

Comment:
I read following: "Вызывает |дляКаждого| для каждого узла для поддерева с родителем == |родитель|. Например, ДляКаждогоВПоддереве(ДайИдентификаторКорня()) вызывает |дляКаждого| для каждого узла включая корень".

The most non-obvious part for me is how parent of a subtree can be it's root? Parent of a subtree is outside the subtree, right? May be you meant |root| here, instead of |parent|?